src/main/kotlin/lotto/DefaultLottoNumber.kt

```diff
@@ -0,0 +1,24 @@
+package lotto
+
+class DefaultLottoNumber private constructor(
+    val numbers: List<Int>,
+) {
+    init {
+        require(numbers.size == DEFAULT_LOTTO_NUMBER_COUNT) { "기본 로또 번호는 6개여야 합니다." }
+        require(numbers.distinct().size == DEFAULT_LOTTO_NUMBER_COUNT) { "중복된 숫자가 있습니다." }
+        require(numbers.all { it in LOTTO_NUMBER_RANGE }) { "범위를 벗어난 숫자가 있습니다." }
```

Comment:
```suggestion
        require(numbers.all { it in LOTTO_NUMBER_RANGE }) { "범위를 벗어난 숫자가 있습니다." }
```
이 클래스는 숫자의 리스트와 관련된 책임을 가지고 있습니다만, 숫자에 대한 책임을 가지고 있지는 않습니다.
프로그래밍 요구사항에는 아래와 같은 항목이 있습니다.
> 일급 컬렉션을 쓴다.
[일급 컬렉션을 사용하는 이유](https://woowacourse.github.io/javable/post/2020-05-08-First-Class-Collection)

일급 컬렉션에 대해서 학습해보시고, 로또 숫자와 관련된 책임을 가지는 클래스를 만들어보면 어떨까요?